src/plonk/verifier.rs

```diff
@@ -13,13 +13,13 @@ use crate::poly::{
 use crate::transcript::{read_n_points, read_n_scalars, EncodedChallenge, TranscriptRead};
 
 /// Returns a boolean indicating whether or not the proof is valid
-pub fn verify_proof<'a, C: CurveAffine, E: EncodedChallenge<C>, T: TranscriptRead<C, E>>(
-    params: &'a Params<C>,
+pub fn verify_proof<'params, C: CurveAffine, E: EncodedChallenge<C>, T: TranscriptRead<C, E>>(
+    params: &'params Params<C>,
     vk: &VerifyingKey<C>,
-    msm: MSM<'a, C>,
+    msm: MSM<'params, C>,
     instance_commitments: &[&[C]],
```

Comment:
Having changed `create_proof` to no longer pass in a `Polynomial` (instead creating it internally and copying in the , we should change this as well to pass in the instance columns directly and then commit to them inside `verify_proof`. Alternatively, if we want to preserve the ability to pass in instance commitments, then we should provide a helper for this conversion, and document its indended usage in the `verify_proof` doc comment.